docs/src/create_reference_md.py

```diff
@@ -6,17 +6,6 @@
 from shared_data_model import DATA_MODEL
 from shared_data_model.meta import Metric, NamedModel, Parameter, Source, Subject
 
-TYPE_DESCRIPTION = {
```

Comment:
Why is this in this PR?

Reply:
ci/quality.sh contained:

```sh
NAMES_TO_IGNORE=''
run vulture --min-confidence 0 --ignore-names $NAMES_TO_IGNORE src/ tests/ .vulture_ignore_list.py
```

Apparently, this caused vulture not to find any unused code because after I changed it to:

```sh
run pipx run `spec vulture` --min-confidence 0 src/ tests/ .vulture_ignore_list.py
```

vulture found the unused `TYPE_DESCRIPTION` dict.

Reply:
Awesome.